### DISCUSSION TITLE
Pime-NG InputOtp  -> startFrom=0,1,-1

### DISCUSSION THREAD

**Discussion by mchuc** [primefaces/community#1550] [PrimeNG]
I would like behavior like: startFromBeginning or startFromEnd

currently you can click on any character field and complete it, but the component itself is nice and designer and I want to use it for the Polish PESEL number, which has 11 digits;
I would like an option that will make the field behave a bit like a regular input wherever I click it, but display the characters in those boxes of the component - but it is completed either from the front (behavior -1), I complete the box I click on - behavior with argument 0, or backfill (behavior with argument 1)

That is:
argument 0 - is as is, the default

`_ _ 1 2 3_ _`
argument -1

no matter where I click, the numbers always fill in from the leftmost one

`1 2 3 _ _ _ _ _
`
argument 1 - if I do not click (at the beginning of entering characters), the numbers are completed from the right

`  _ _ __ _ 1 2 3`

**Comment by mchuc** [comment]
You could also add the option "complete with zeros", e.g. for date of birth or time

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the feedback we'll wait for upvotes to implement this feature.
